libr/core/cio.c

```diff
@@ -139,7 +139,7 @@ R_API int r_core_write_op(RCore *core, const char *arg, char op) {
 				eprintf ("Clipboard is empty and no value argument(s) given\n");
 				goto beach;
 			}
-			str = r_mem_dup (r_buf_buffer (core->yank_buf), len);
+			str = r_mem_dup ((ut8*)r_buf_buffer (core->yank_buf), len);
```

Comment:
i think you should change r_mem_dup to accept `const void *` instead of forcing the casting here.

Reply:
In fact there are other consumers in other places.

Reply:
What do you mean? Changing the API to use `const void *` instead of `void *` should not be a problem.